samples/remote_attestation/common/dispatcher.cpp

```diff
@@ -182,10 +182,177 @@ int ecall_dispatcher::verify_report_and_set_pubkey(
     return ret;
 }
 
+int ecall_dispatcher::establish_secure_channel(uint8_t** key, size_t* key_size)
+{
+    uint8_t encrypted_key_buf[512];
+    uint8_t signature[256];
+    uint8_t digest[32];
+    size_t encrypted_key_size = 256;
+    size_t signature_size = 256;
```

Comment:
Does this signature_size has to be 256? 
Would 32 be enough?

Reply:
also, it would be great to define constants for those numbers (32, 256, 512 used above) to make it easier to read the code.

Reply:
Done